src/maincurrencydialog.cpp

```diff
@@ -150,7 +152,7 @@ void mmMainCurrencyDialog::CreateControls()
     itemBoxSizer22->Add(update_button, g_flagsH);
     update_button->Connect(wxID_STATIC, wxEVT_COMMAND_BUTTON_CLICKED
         , wxCommandEventHandler(mmMainCurrencyDialog::OnOnlineUpdateCurRate), nullptr, this);
-    update_button->SetToolTip(_("Online update currency rate"));
+    update_button->SetToolTip(_("Online update rates for all used currencies"));
```

Comment:
We should mention that only current (today/yesterday) rates will be updated using this button